rosbag2_performance/writer_benchmarking/include/byte_producer.hpp

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020, Robotec.ai sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef BYTE_PRODUCER_HPP_
```

Comment:
Actually, now it should be ROSBAG2_PERFORMAMCE_WRITER_BENCHMARKING__BYTE_PRODUCER_HPP_ i believe (with package name changed according to your suggestion) - i will apply this

Reply:
Linter does complain - it demands compliance with class name it seems. I have changed it according to your suggestion nevertheless. These includes aren't exported anywhere, so I guess it could stay either way.